etherpad.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+    # Etherpad: real-time collaborative document editing
+    etherpad:
+        image: jitsi/etherpad
+        networks:
+            meet.jitsi:
```

Comment:
add an alias here so we can make it work automatically

Reply:
@saghul, sorry I don't understand what do you mean. It's already works automatically because we set ETHERPAD_URL_BASE=http://etherpad:9001 and the service `etherpad` visible in network `meet.jitsi`

Reply:
Yeah, you;re right, sorry.